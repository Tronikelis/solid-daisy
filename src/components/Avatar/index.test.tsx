import { render } from "@solidjs/testing-library";
import { describe, expect, it } from "vitest";

import { Avatar } from ".";

describe("<Avatar />", () => {
    it("renders correctly", () => {
        const { container } = render(() => <Avatar src="https://example.com" />);
        expect(container).toMatchSnapshot();
    });
});
