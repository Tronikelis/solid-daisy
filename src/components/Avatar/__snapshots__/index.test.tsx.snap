// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`<Avatar /> > renders correctly 1`] = `
<div>
  <div
    class="flex-shrink-0 overflow-hidden rounded-full *:aspect-square size-16"
  >
    <img
      class="object-fit h-full w-full"
      loading="lazy"
      referrerpolicy="no-referrer"
      src="https://example.com"
    />
  </div>
</div>
`;
